core/site.go

```diff
@@ -95,6 +95,9 @@ type Site struct {
 	batterySoc   float64         // Battery soc
 	batteryMode  api.BatteryMode // Battery mode
 
+	// user settings
+	language string // Language
```

Comment:
Bin ich jetzt kein so großer Fan von. Muss das im Backend sein? Und falls ja: dann sollte es sich vmtl. auch auf den CSV Export auswirken?

Reply:
Wobei das eher locale denn Sprache wäre…

Reply:
> Und falls ja: dann sollte es sich vmtl. auch auf den CSV Export auswirken?

Ja, aktuell wird beim CSV Aufruf die Sprache explizit übergeben. Das könnte man durch die eingestellte Sprache ersetzen. Allerdings bräuchte der Endpunkt dann noch ne Logik, um den "automatisch" (default) Fall zu handeln. Momentan bekommt der immer einer valide Sprache übergeben.

Daher habe ich das in diesem Schritt noch nicht angefasst und hab erstmal die existierende Logik für CSV gelassen.

Reply:
Locale vs. Language: Hier gehts wirklich nur um Sprache, so wie wir sie in Weblate abbilden.